src/components/List/List.tsx

```diff
@@ -23,8 +24,12 @@ export const List: FC<IGenreListProps> = ({ itemNames, preselectedItemName, onSe
         onSelect(itemName);
     }
 
+    const listClassNames : string[] = ["list"];
```

Comment:
probably better to use `useState`

Reply:
but it is not state of component, it's just class name